org.flatpak.ClassicSnakeTerminal.yml

```diff
@@ -0,0 +1,50 @@
+app-id: org.flatpak.ClassicSnakeTerminal
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
```

Comment:
does this also work with 20.08? Whilst 19.08 is still within security support the more recent release will be for longer.